pkgs/top-level/haskell-packages.nix

```diff
@@ -49,13 +49,13 @@ in {
 
     ghc822 = callPackage ../development/compilers/ghc/8.2.2.nix {
       bootPkgs = packages.ghc822Binary;
-      inherit (buildPackages.python3Packages) sphinx;
+      sphinx = buildPackages.python3Packages.sphinx_1_7_9;
```

Comment:
Why did you make that change? ghc-8.2.2 builds fine at the moment, unlike ghc-8.4.4, so this feels unnecessary. Is there any particular reason?

Reply:
I just assumed the sphinx breakage was only fixed in >= ghc8.6.